pibooth/view/background.py

```diff
@@ -90,7 +90,7 @@ def _write_text(self, text, rect=None, align='center'):
             rect = self._rect.inflate(-self._text_border, -self._text_border)
         if self._outlines:
             # Build rectangle around text area for debuging purpose
-            outlines = pygame.Surface(rect.size)
+            outlines = pygame.Surface(rect.size, pygame.SRCALPHA, 32)
```

Comment:
Can you tell me what is this change? I tried with and without it and can't see the diff...

Reply:
Use the debug mode with a background image, and admire the improvement ;-)